src/styles/main.scss

```diff
@@ -1,3 +1,37 @@
+$padding-big: 128px;
+$padding-medium: 40px;
+$horizontal-spacing: 50px;
+
+@mixin horizontal-paddings ($size) {
+  padding-left: $size;
+  padding-right: $size;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  margin-bottom: $horizontal-spacing;
+}
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  @include horizontal-paddings($padding-medium);
```

Comment:
![Screenshot 2020-07-20 at 11 26 27](https://user-images.githubusercontent.com/35385361/87916462-2395d980-ca7c-11ea-97a0-6f35c0b0d584.png)
max-width can help you with this, it would be great if on big screens users got to look on 4 cards per line